src/dpr/components/report-list/data/reportingClient.ts

```diff
@@ -0,0 +1,60 @@
+import ReportQuery from '../../../types/ReportQuery'
+import logger from '../../../utils/logger'
+import RestClient from './restClient'
+import Dict = NodeJS.Dict
+import { components, operations } from '../../../types/api'
+import { ApiConfig } from './types'
+
+export interface Count {
+  count: number
+}
+
+export default class ReportingClient {
+  restClient: RestClient
+
+  constructor(config: ApiConfig) {
+    this.restClient = new RestClient('Reporting API Client', config)
+  }
+
+  getCount(resourceName: string, token: string, listRequest: ReportQuery): Promise<number> {
```

Comment:
minor: Should the listRequest param here be called countRequest instead?

Reply:
Fixed